pepper-apis/parent-pom.xml

```diff
@@ -406,6 +405,12 @@
             <groupId>org.quartz-scheduler</groupId>
             <artifactId>quartz</artifactId>
             <version>2.3.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.zaxxer</groupId>
+                    <artifactId>HikariCP-java6</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Quartz comes with an old version of HikariCP (for java 6 !) which conflicts with what we declared. Excluding it so we get the right version into the jar. We're not using database persistence of jobs in Quartz so this should matter to Quartz functionality.